R/limits.r

```diff
@@ -4,16 +4,12 @@
 #' individual scales. Note that, by default, any values outside the limits
 #' will be replaced with `NA`.
 #'
-#' @param ... A name-value pair. The name must be an aesthetic, and the value
-#'   must be either a length-2 numeric, a character, a factor, or a date/time.
+#' @param ... For `xlim` and `ylim`: Two numeric values, specifying the left/lower limit and the right/upper limit of the scale. If the larger value comes first, the scale will be reversed. You can leave one value as NA to compute from the range of the data.
```

Comment:
Could you please manually word-wrap the text so that individual lines fit onto the screen? And also, for all lines but the one starting with `@param`, please add two spaces of indentation.